src/lib/Hydra/Controller/User.pm

```diff
@@ -143,6 +157,38 @@ sub doEmailLogin {
 }
 
 
+sub oidc_login :Path('/oidc-login') Args(0) {
+    my ($self, $c) = @_;
+
+    error($c, "Logging in via OIDC is not enabled.", 404) unless $c->config->{enable_oidc_login};
```

Comment:
Why not scope these? Something like `$c>config->{oidc}->{enable}`? Or even better `$c->config->{oidc}->{client_id}` which can be undef or defined and enable/disable oidc auth that way